crates/aptos-telemetry-service/src/context.rs

```diff
@@ -7,19 +7,48 @@ use crate::{
         victoria_metrics_api::Client as MetricsClient,
     },
     types::common::EpochedPeerStore,
+    MetricsEndpointsConfig,
 };
 use aptos_crypto::{noise, x25519};
 use aptos_infallible::RwLock;
 use aptos_types::{chain_id::ChainId, PeerId};
 use jsonwebtoken::{Algorithm, DecodingKey, EncodingKey, TokenData, Validation};
 use serde::{de::DeserializeOwned, Serialize};
 use std::{
-    collections::{BTreeMap, HashMap, HashSet},
+    collections::{HashMap, HashSet},
     convert::Infallible,
     sync::Arc,
 };
 use warp::Filter;
 
+#[derive(Clone, Default)]
+pub struct GroupedMetricsClients {
+    pub telemetry_service_metrics: HashMap<String, MetricsClient>,
```

Comment:
We could use a comment explaining the purpose of different types of metrics.

Reply:
Yes, I have addressed it.